### PR TITLE
Update the documentation to reflect the new commands and options

### PR DESCRIPTION
**Pull request by abertelrud**
This updates the documentation in the swiftpm repository to reflect the updated subcommand names and option names. A separate PR will do the same for the swift.org web page.

### CODE REVIEW

File: Documentation/SystemModules.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ Lets put this code in a directory called `example`:
 
     $ mkdir example
     $ cd example
-    example$ touch main.swift Package.swift
+    example$ swift package init --type exec
+    example$ touch main.swift
```
Comment (ddunbar) [inline Documentation/SystemModules.md]: `touch main.swift` is wrong here, the init command creates the source file (but I believe creates it in Sources, we should make sure these instructions work)

File: Documentation/SystemModules.md [hunk 2]
```diff
@@ -98,7 +101,8 @@ First create a directory called `CJasPer` parallel to `CJPEG` and our example ap
     CJPEG$ cd ..
     $ mkdir CJasPer
     $ cd CJasPer
-    CJasPer$ touch module.modulemap Package.swift
+    CJasPer$ swift package init
```
Comment (ddunbar) [inline Documentation/SystemModules.md]: This isn't correct, `swift package init` will add other files which are not appropriate in a system module map package.

Comment (abertelrud) [inline Documentation/SystemModules.md]: So isn't the real way to fix this to add "system-module" (or something like that) to the `init` subcommand's --type flag?

File: Documentation/SystemModules.md (hunk 1 above)
Comment (abertelrud) [inline Documentation/SystemModules.md]: Thanks.  Yes, testing out the instructions is part of the pre-merge testing I haven't done yet.

File: Documentation/SystemModules.md (hunk 2 above)
Comment (ddunbar) [inline Documentation/SystemModules.md]: Yes, probably...

File: Documentation/SystemModules.md (hunk 1 above)
Comment (binh045) [inline Documentation/SystemModules.md]: A

File: Documentation/SystemModules.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ Lets put this code in a directory called `example`:
 
     $ mkdir example
     $ cd example
-    example$ touch main.swift Package.swift
+    example$ swift package init --type executable
+    example$ touch main.swift
     example$ open -t main.swift Package.swift
 
 To `import CJPEG` the package manager requires
```
Comment (binh045) [inline Documentation/SystemModules.md]: A

File: README.md [hunk 4]
```diff
@@ -37,16 +37,16 @@ The package manager is bundled with the [**Trunk Development** Snapshots availab
 
         export PATH=path/to/toolchain/usr/bin:$PATH
 
-You can verify your installation by typing `swift build --version` in a terminal:
+You can verify your installation by typing `swift package --version` in a terminal:
```
Comment (binh045) [inline README.md]: Trr

File: README.md [hunk 5]
```diff
@@ -37,16 +37,16 @@ The package manager is bundled with the [**Trunk Development** Snapshots availab
...
 
 ```sh
```
Comment (binh045) [inline README.md]: Sh

**Pull request merged**